server.js

```diff
@@ -25,13 +26,17 @@ app.prepare().then(() => {
 
     if (req.headers.host === 'man.dvc.org') {
       let normalized_pathname =
-        pathname !== '/import-url' ? pathname.replace(/-/i, '/') : pathname
+        ['/get-url', '/import-url'].indexOf(pathname) > 0
+          ? pathname.replace(/-/i, '/')
+          : pathname
       const doc_pathname = '/doc/commands-reference' + normalized_pathname
       res.writeHead(301, { Location: 'https://dvc.org' + doc_pathname })
       res.end()
     } else if (req.headers.host === 'remote.dvc.org') {
       res.writeHead(301, {
-        Location: 'https://s3-us-west-2.amazonaws.com/dvc-storage' + pathname
+        Location:
+          'https://s3-us-west-2.amazonaws.com/dvc-public/remote/' +
```

Comment:
most likely it should be us-east something address

Reply:
Yes, `us-east-2`, thanks. I guess `dvc-share` was in `us-west-2` and I just updated the ending of the path. But it's already been updated by now (: